skbio/tree/_majority_rule.py

```diff
@@ -124,10 +124,14 @@ def _build_trees(clade_counts, edge_lengths, support_attr):
         Keyed by the frozenset of the clade and valued by the weighted length
     support_attr : str
         The name of the attribute to hold the support value
+    tree_node_class : Python class
```

Comment:
It can be any class that has the same interface as TreeNode (though in most cases will be a subclass). Note that this parameter is different than `TreeNode.from_taxonomy/from_linkage_matrix` because those require `TreeNode` or subclass since they are `classmethods`.